client/components/lessons/LessonNavigation/LessonNavigation.module.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../../styles/variables";
+
+.lesson-navigation {
+  @extend %containerConfig;
+  display: flex;
+  gap: 1em;
+  flex-direction: column;
+  padding: 0.75em 0;
+
+  > section {
+    flex: 1;
+
+    &:not(:only-child):first-child {
```

Comment:
I left a comment explaining what happens. In my opinion, it's better to only select the targetted element where possible, instead of selecting multiple one and then overriding the styling.
If you still think it's hard to read, I can modify it though.